pkgs/applications/networking/modem-manager-gui/remove-fhs-gdbm-check.patch

```diff
@@ -0,0 +1,16 @@
+--- a/configure	2018-10-25 14:19:33.361529316 +1100
```

Comment:
This should no longer be required with meson.

Reply:
fixed